packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -404,9 +404,11 @@ export class DataStores implements IDisposable {
                 const contextSummary = await context.summarize(fullTree, trackState);
                 summaryBuilder.addWithStats(contextId, contextSummary);
 
-                // Prefix the child's id to the ids of its GC nodest. This gradually builds the id of each node to
-                // be a path from the root.
-                gcDataBuilder.prefixAndAddNodes(contextId, contextSummary.gcData.gcNodes);
+                if (contextSummary.gcData !== undefined) {
```

Comment:
I added this so that when we remove gcData from summarize, it becomes a non-breaking change and we don't have to wait another couple of versions to remove it.